File: nysol_python/mcmd/methods/convTex/db/smp/mnewstr.py
# ================================================
# コマンドマニュアル自動作成用サンプルコードデータ
# ================================================
db={}
db['name']='mnewstr'

############################### IDATA
db['idatas']=[]

############################### SCRIPTS
db['scripts']=[]

script={}
script['title']='基本例'
script['comment']='''
``custNo`` と ``A0001`` という文字列データを5行作成し、 ``attribute,code`` という名前の項目名で出力する。
'''
script['sh_code']='''
mnewstr a=attribute,code v=custNo,A0001 l=5 o=rsl1.csv
'''
script['py_code']='''
nm.mnewstr(a="attribute,code", v="custNo,A0001", l="5", o="rsl1.csv").run()
'''
script['odatas']='rsl1.csv'
db['scripts'].append(script)

File: olddoc/mcmd/en/examples/mnewstr.rb
#!/usr/bin/env ruby
# coding: utf-8
require "./mkTex.rb"

############## Example 1
title="Basic Example"
comment=<<'EOF'
Generate a new dataset with characters strings \verb|custNo| and \verb|A0001| printed in 5 rows, and name the fields as \verb|attribute| and \verb|code| respectively.
EOF
scp=<<'EOF'
mnewstr a=attribute,code v=custNo,A0001 l=5 o=rsl1.csv
more rsl1.csv
EOF
run(scp,title,comment)
